docs/source/torch.rst

```diff
@@ -625,10 +625,14 @@ Symbolic Numbers
     :toctree: generated
     :nosignatures:
 
+    SymInt
```

Comment:
These don't work https://docs-preview.pytorch.org/92149/torch.html#symbolic-numbers
I think you need a `.. autoclass:: SymInt` here like it is done for Tag below.